### PR TITLE
Update PoolInforCard

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?

- update the `PoolInfoCard.tsx`.
- Remove the `PoolInfoCard.scss`.

## Context

https://www.figma.com/file/Aa3IfDUO68aCNr8nZTuQ8S/Product-Design-with-MUI?node-id=106%3A12066
close: https://github.com/saddle-finance/saddle-frontend/issues/948

## Screenshot
Screen Shot 2022-04-16 at 1 59 35 PM
Screen Shot 2022-04-16 at 1 59 47 PM
Screen Shot 2022-04-16 at 2 00 10 PM
Screen Shot 2022-04-16 at 2 00 49 PM

### CODE REVIEW

File: src/components/PoolInfoCard.tsx [hunk 1]
```diff
@@ -61,73 +69,96 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
...
+
+        <InfoItem>
+          <Typography>{`${t("utilization")}:`}</Typography>
+          <Typography variant="subtitle1">
+            {formattedData.utilization}
+          </Typography>
+        </InfoItem>
+      </Box>
+      <InfoItem>
+        <Typography>{`${t("adminFee")}:`}</Typography>
```
Comment (ug02fast) [inline src/components/PoolInfoCard.tsx:136]:
```suggestion
        <Typography>{`${t("adminFee")}: `}</Typography>
```
There needs to be a space after colon.

File: src/components/PoolInfoCard.tsx [hunk 2]
```diff
@@ -61,73 +69,96 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
...
-          </div> */}
-        </div>
-      </div>
-      <Divider />
-      <div className="bottom">
-        <h4>{t("currencyReserves")}</h4>
-        <span>{`$${formattedData.reserve} ${t("inTotal")}`}</span>
-        <div className="tokenList">
+          <Typography variant="subtitle1">
+            :{formattedData.aParameter}
```
Comment (ug02fast) [inline src/components/PoolInfoCard.tsx:124]: Why is it for this occurrence the colon is where the value is? There also needs to be a space. Can you move it to the above field?

File: src/components/PoolInfoCard.tsx [hunk 3]
```diff
@@ -61,73 +69,96 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
...
+          data?.isPaused ? t("paused") : t("active")
+        }`}</Typography>
+      </InfoItem>
+      <InfoItem>
+        <Typography>{`${t("totalLocked")}:`}</Typography>
+        <Typography variant="subtitle1">{`$${formattedData.reserve}`}</Typography>
+      </InfoItem>
+      {/* <InfoItem>
+        <Typography>{t("dailyVolume") + ": "}</Typography>
+        <Typography variant="subtitle1">{formattedData.volume}</Typography>
```
Comment (ug02fast) [inline src/components/PoolInfoCard.tsx:101]: Why is this commented out? Do we have volume data?

**Review by ug02fast — CHANGES_REQUESTED** [review]
image
Also this needs to be LP Amount.

Please double check with Figma on the design of the smaller details.

File: src/components/PoolInfoCard.tsx [hunk 4]
```diff
@@ -61,73 +69,96 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
...
-        <div className="infoItem">
+      <InfoItem mt={3}>
+        <Typography>{`${t("status")}:`}</Typography>
+        <Typography variant="subtitle1">{`${
+          data?.isPaused ? t("paused") : t("active")
+        }`}</Typography>
+      </InfoItem>
+      <InfoItem>
+        <Typography>{`${t("totalLocked")}:`}</Typography>
+        <Typography variant="subtitle1">{`$${formattedData.reserve}`}</Typography>
```
Comment (ug02fast) [inline src/components/PoolInfoCard.tsx:97]:
```suggestion
        <Typography>{`${t("totalLocked")}: `} <Typography variant="subtitle1">{`${formattedData.reserve}`}</Typography></Typography>
```

Can you see if the formatting works fine with this structure? Also this had an extra `$`

**Review by ug02fast — CHANGES_REQUESTED** [review]

File: src/components/PoolInfoCard.tsx (hunk 2 above)
Comment (nkrulikovsky) [reply src/components/PoolInfoCard.tsx:124]: Because tooltop is wrapping the colon as well.

File: src/components/PoolInfoCard.tsx (hunk 1 above)
Comment (nkrulikovsky) [reply src/components/PoolInfoCard.tsx:136]: Oh, I didn't check that there is space of 4px between property name and value

File: src/components/TopMenu.tsx [hunk 5]
```diff
@@ -111,7 +111,7 @@ function TopMenu(): ReactElement {
             <NetworkDisplay onClick={handleSettingMenu} />
             <IconButton
               onClick={handleSettingMenu}
-              data-testId="settingsMenuBtn"
+              data-testid="settingsMenuBtn"
```
Comment (ug02fast) [inline src/components/TopMenu.tsx:114]: nice

**Review by ug02fast — APPROVED** [review]

**Comment by hammeiam** [comment]
Some notes after looking at the screenshots:
- My Share USD balance should have a `$` symbol somewhere. How we represent USD should be consistent throughout the app, but it's shown in 3 different ways different between My Share, Total Locked, and Currency Reserves. 
- Something is wrong with the decimal precision of the Currency Reserves; note how the total is $173m but each asset is only in the thousands.
- The spacing of Fee and Utilization is strange, I'm not sure why they're so far to the right in the desktop layout.

**Comment by hammeiam** [comment]
Could you add a style to the token icons that makes them all the same size? It looks like lpTokens are showing up a bit larger, though I don' think that was because of this PR. Also could you fix the desktop spacing of `Utilization` in the pool info panel?

Screen Shot 2022-04-13 at 12 45 55 PM
Screen Shot 2022-04-13 at 12 45 51 PM
Screen Shot 2022-04-13 at 12 45 45 PM

**Comment by nkrulikovsky** [comment]
@hammeiam 
Good call out!
Gave default width and height of 24 on `TokenIcon`.
And on the design we put pool infos on two columns, but actually, virtual price is having long width than design, so it overflow.
So, my suggestion is to put pool infos on one column.

File: src/components/PoolInfoCard.tsx [hunk 6]
```diff
@@ -106,37 +101,35 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
...
+          return formattedData.aprs[key]?.apr.gt(Zero) ? (
+            <InfoItem key={symbol}>
+              {symbol.includes("/") ? (
+                <Tooltip title={symbol.replaceAll("/", "\n")}>
+                  <Typography sx={{ borderBottom: "1px dotted" }}>
+                    Reward APR:
+                  </Typography>
+                </Tooltip>
+              ) : (
+                <Typography>{symbol} APR: &nbsp;</Typography>
```
Comment (ug02fast) [inline src/components/PoolInfoCard.tsx:124]: Does using template literals: ` ${} ` here with a space work so you don't have to use &nbsp;?

It'd probably be `${symbol} APR: ` but with the backticks.

File: src/components/PoolInfoCard.tsx [hunk 7]
```diff
@@ -181,14 +174,19 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
...
+                  width={20}
+                  height={20}
+                />
                 <Typography
                   variant="subtitle1"
                   ml={1}
                 >{`${token.symbol} ${token.percent}`}</Typography>
               </Box>
-              <Typography className="tokenValue">${token.value}</Typography>
+              <Typography data-testid="tokenValue">{token.value}</Typography>
```
Comment (ug02fast) [inline src/components/PoolInfoCard.tsx:189]: Can you make this more unique to the component? This is where we will start having clashing of data-testid names. 

Maybe something like `poolInfoCardTokenValue`

File: src/components/WithdrawPage.tsx [hunk 8]
```diff
@@ -225,13 +225,7 @@ const WithdrawPage = (props: Props): ReactElement => {
...
           </Button>
         </Box>
-        <Stack
-          direction="column"
-          flex={1}
-          spacing={4}
-          width="100%"
-          maxWidth="50%"
-        >
+        <Stack direction="column" flex={1} spacing={4} width="100%">
```
Comment (ug02fast) [inline src/components/WithdrawPage.tsx:228]: Do you always have to specify width 100%? I feel like if parent is flexing then children elements or containers should automatically fill all width?

**Review by ug02fast — CHANGES_REQUESTED** [review]

File: src/components/WithdrawPage.tsx (hunk 8 above)
Comment (nkrulikovsky) [reply src/components/WithdrawPage.tsx:228]: You are right!

File: src/components/PoolInfoCard.tsx (hunk 7 above)
Comment (nkrulikovsky) [reply src/components/PoolInfoCard.tsx:189]: Oh, right now this component, don't need testid actually. removed

File: src/components/PoolInfoCard.tsx (hunk 6 above)
Comment (nkrulikovsky) [reply src/components/PoolInfoCard.tsx:124]: Yes, it would be more pretty clear!

File: src/components/PoolInfoCard.tsx [hunk 9]
```diff
@@ -28,13 +28,10 @@ const InfoItem = styled(Box)(({ theme }) => ({
   display: "flex",
   alignItems: "center",
   flexDirection: "row",
-  minWidth: "50%",
+  minWidth: "100%",
```
Comment (ug02fast) [inline src/components/PoolInfoCard.tsx:31]: what happens is minWidth is deleted here? does the style break?

**Review by ug02fast — APPROVED** [review]
lgtm

File: src/components/PoolInfoCard.tsx (hunk 9 above)
Comment (nkrulikovsky) [reply src/components/PoolInfoCard.tsx]: I have added that property for putting 2 Item info on one row.
But in the case we put a Item in one row, We can remove `minWidth`

File: src/components/PoolInfoCard.tsx [hunk 10]
```diff
@@ -1,24 +1,38 @@
...
-import { formatBNToPercentString, formatBNToString } from "../utils"
-import { Divider } from "@mui/material"
+import { commify, formatUnits } from "@ethersproject/units"
+import {
+  formatBNToPercentString,
+  formatBNToShortString,
+  formatBNToString,
+} from "../utils"
+
+// import { Partners } from "../utils/thirdPartyIntegrations"
```
Comment (hammeiam) [inline src/components/PoolInfoCard.tsx:16]: rm comment

**Review by hammeiam — APPROVED** [review]

**Pull request merged**